mega-zep-backend/src/main/java/com/gepardec/mega/application/configuration/ZepConfig.java

```diff
@@ -17,7 +17,15 @@
     @ConfigProperty(name = "mega.zep.url")
     URL url;
 
+    @Inject
```

Comment:
@Ruhsi   
Don't forget to enhance the test class 'ConfigResourceTest' for the newly introduced parameters.

Reply:
@cchet the soap path is not need in the Frontend. So not needed in ConfigResource(Test).

Reply:
@Ruhsi  We need to test if the config model is filled properly with the values from the configuration provided to the application. In fact it's just a assert in the single test method thetest class has. It ensures that the configuration provided to the application is properly set in the response model, whereby the url is build by the origin + path configuration properties.

Reply:
@cchet we only send the origin to the frontend so we cannot test it in the config endpoint response. The soap path is only internally used by the backend